aperturedb/sqlc/migrations/000003_lnc-sessions.up.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS sessions (
+    id INTEGER PRIMARY KEY,
+
+    passphrase_entropy BLOB NOT NULL UNIQUE,
```

Comment:
Is this the raw bytes, or the string pass phrase?

Reply:
added comments to all fields